MAINTAINERS

```diff
@@ -138,10 +138,10 @@ W: http://plugins.geany.org/geanyprj.html
 S: Odd Fixes
 
 geanypy
-P: Lex Trotman <elextr@gmail.com>
-M: Lex Trotman <elextr@gmail.com>
```

Comment:
Let's get technical and review the patch :-).

I actually agree with the "-" part of the patch - having GP version of the plugin maintained by someone else than the upstream author is less than ideal. IMO it would be best having Matthew Brush at the "+" side of the patch if @codebrainz is willing to maintain it. 

Otherwise I think it doesn't make sense to keep an unmaintained copy of a maintained plugin in GP.